docs/src/RunningOnGPUs.md

```diff
@@ -0,0 +1,45 @@
+# Running on GPUs
+
+ITensor provides package extensions for running tensor operations in the GPU with different backends.
+You can activate a backend by loading the Julia GPU package alongside ITensors.jl
```

Comment:
It would be helpful to clarify that you mean a vendor-specific GPU package, since "the Julia GPU package" sounds like it is a specific, single package. How about "by loading the appropriate GPU package" ?